src/dev/build/tasks/bundle_fleet_packages.ts

```diff
@@ -28,9 +28,10 @@ export const BundleFleetPackages: Task = {
     // Support the `--use-snapshot-epr` command line argument to fetch from the snapshot registry
     // in development or test environments
     const { buildOptions } = readCliArgs(process.argv);
-    const eprUrl = buildOptions?.useSnapshotEpr
-      ? 'https://epr-snapshot.elastic.co'
-      : 'https://epr.elastic.co';
+    const eprUrl =
```

Comment:
When this is merged we'll have to update the release manager arguments in infra to use production, I can assist if needed.

Reply:
We don't want to use production actually, we want to use snapshot for the time being. So I don't think we need any adjustments right now.